vllm/worker/model_runner.py

```diff
@@ -523,15 +535,17 @@ def _pad_to_max(x: List[int], max_len: int, pad: int) -> List[int]:
     return x + [pad] * (max_len - len(x))
 
 
-def _make_tensor_with_pad(
-    x: List[List[int]],
-    max_len: int,
-    pad: int,
-    dtype: torch.dtype,
-    device: Union[str, torch.device] = "cuda",
-) -> torch.Tensor:
+def _make_tensor_with_pad(x: List[List[int]],
+                          max_len: int,
+                          pad: int,
+                          dtype: torch.dtype,
+                          device: Union[str, torch.device] = "cuda",
+                          pin_memory=False) -> torch.Tensor:
```

Comment:
```suggestion
                          pin_memory=False,) -> torch.Tensor:
```
Could you append `,` here so that we can minimize the code change?

Reply:
Done.